src/surface.lisp

```diff
@@ -11,8 +11,12 @@
   (c-ref surface sdl2-ffi:sdl-surface :pixels))
 
 (defun surface-format (surface)
+  (c-ref surface sdl2-ffi:sdl-surface :format))
+
+(defun surface-format-format (surface)
   (enum-key '(:enum (sdl-pixel-format))
-            (c-ref surface sdl2-ffi:sdl-surface :format :format)))
+            (c-ref (c-ref surface sdl2-ffi::sdl-surface :format)
+                   sdl2-ffi::sdl-pixel-format :format)))
 
```

Comment:
You should be able to just say `(c-ref ... :format :format)` here.  (Note, get the latest cl-autowrap.)  I'm not sure it's hugely necessary to have a separate function to get the pointer, since most things take the value.